pkg/tenant/auth.go

```diff
@@ -199,6 +206,13 @@ func (s *Server) Refresh(c *gin.Context) {
 		return
 	}
 
+	// Set the access and refresh tokens as cookies for the front-end
+	if err := middleware.SetAuthTokens(c, reply.AccessToken, reply.RefreshToken, s.conf.Auth.CookieDomain); err != nil {
+		sentry.Error(c).Err(err).Msg("could not set access and refresh token cookies")
+		c.JSON(http.StatusInternalServerError, api.ErrorResponse("could not set auth cookies"))
+		return
+	}
+
```

Comment:
Set the cookies in Login and Refresh -- anywhere else?

Reply:
This is just Tenant, which is the only thing that has a frontend; should we also set it in Quarterdeck for future proofing? Will we ever have a UI that goes directly to Quarterdeck?